cluster/validation.go

```diff
@@ -67,13 +67,59 @@ func validateAuthOptions(c *Cluster) error {
 	return nil
 }
 
+func transformAciNetworkOption(option string) string {
+	switch option {
+	case AciSystemIdentifier:
+		option = "system_id"
+	case AciServiceGraphSubnet:
+		option = "node_svc_subnet"
+	case AciStaticExternalSubnet:
+		option = "extern_static"
+	case AciDynamicExternalSubnet:
+		option = "extern_dynamic"
+	}
+	return option
+}
+
 func validateNetworkOptions(c *Cluster) error {
-	if c.Network.Plugin != NoNetworkPlugin && c.Network.Plugin != FlannelNetworkPlugin && c.Network.Plugin != CalicoNetworkPlugin && c.Network.Plugin != CanalNetworkPlugin && c.Network.Plugin != WeaveNetworkPlugin {
+	if c.Network.Plugin != NoNetworkPlugin && c.Network.Plugin != FlannelNetworkPlugin && c.Network.Plugin != CalicoNetworkPlugin && c.Network.Plugin != CanalNetworkPlugin && c.Network.Plugin != WeaveNetworkPlugin && c.Network.Plugin != AciNetworkPlugin {
 		return fmt.Errorf("Network plugin [%s] is not supported", c.Network.Plugin)
 	}
 	if c.Network.Plugin == FlannelNetworkPlugin && c.Network.MTU != 0 {
 		return fmt.Errorf("Network plugin [%s] does not support configuring MTU", FlannelNetworkPlugin)
 	}
+	if c.Network.Plugin == AciNetworkPlugin {
+		//Skipping Cloud options for now
```

Comment:
Does `for now` mean in the end commit or is this subject to change? if this is going to be committed, it needs a note why this is skipped and what the plan is

Reply:
I have to do more testing with this template in cloud environment before I add validation. I will make another commit when I test cloud version.

Reply:
As discussed, added validation to disable cloud options. Please continue your review